microsoft-mxchip/README.md

```diff
@@ -18,49 +18,20 @@ The audio data is sent via the 3.5mm jack output.
 
 ## Requirements
 
-You will need a Chirp developer account to use the SDK, which you can sign up for at the [Chirp developer hub](https://developers.chirp.io). After signing up, you'll have a Chirp application with a key and secret, and should be able to download the config file (.conf) for 16kHz audio sample rates (`16kHz` protocol in your Chirp account). The downloaded file should be named `16kHz.conf`.
+- Follow the [official IoT DevKit tutorial](https://microsoft.github.io/azure-iot-developer-kit/docs/get-started/) to get set up. This guide will help you install all the toolchains needed. Follow the steps up until the `ST-Link configuration` section.
 
-You also need to have a working development environment for your MXChip. You can follow the [official IoT DevKit tutorial](https://microsoft.github.io/azure-iot-developer-kit/docs/get-started/). This will help you install all the toolchains needed. Follow the steps up until the `ST-Link configuration` section.
+- Sign up to Chirp at the [Chirp developer hub](https://developers.chirp.io).
+- Install ChirpSDK through the Arduino library manager. This can also be accessed via Visual Studio.
 
-## Adding the Chirp SDK to the Arduino build process
+## Setup
 
-Unfortunately, the Arduino toolchain doesn't offer any quick and easy way to add custom archive libraries to Arduino projects. The file needs to be added manually by following these steps:
+Copy/paste your Chirp app key, secret and config string for the `16kHz` protocol into the `credentials.h` file.
+Currently only the `16kHz` protocol will work on embedded devices.
```

Comment:
If we update the lib this won't be true anymore X)

Reply:
and 16kHz mono?

Reply:
yeah at least this one